### PR TITLE
Added ros1/simulators-osrf image

### PR DESCRIPTION
**Pull request by peci1**
I know you suggest to not add new ROS 1 images, but this one could hopefully be an exception :) Not sure if the same with desktop-full is desired...

### CODE REVIEW

**Review by sloretz — COMMENTED** [review]
I'm good with adding this image!

Mind also adding it to the table in the README under Noetic with maybe a note below the table describing it?

The Noetic action is pretty long and I had some trouble early on running out of disk space, so I want to try running this one in an action before I merge the PR. It will be a few days to before I get some time to try that though, so thanks in advance for your patience!

File: scripts/test_images.py [hunk 1]
```diff
@@ -98,7 +99,7 @@ def main():
             full_name = _full_name(args.registry, args.name, tag)
             for arch, variant in architectures:
                 if arch == "arm" and variant == "v7":
-                    if s in ("simulators", "desktop-full"):
+                    if s in ("simulators", "simulators-osrf", "desktop-full"):
```
[the remark this answers is not in the document]
Comment (peci1) [reply scripts/test_images.py:102]: Good catch, resolved.

**Comment by peci1** [comment]
Thanks, I'll wait as needed, no rush from my side ;)

> Mind also adding it to the table in the README under Noetic with maybe a note below the table describing it?

Added.

**Comment by sloretz** [comment]
Run in progress: https://github.com/sloretz/ros_oci_images/actions/runs/8556334203

**Comment by sloretz** [comment]
Looks like the job succeeded 🎉 @peci1 mind running the `simulators-osrf` image from https://github.com/sloretz/ros_oci_images/pkgs/container/ros-peci1-gz-osrf and making sure it has a newer gazebo version than the `simulators` package? If it looks good to you I'll merge this PR.

**Comment by peci1** [comment]
Yes, it works!

```
$ docker run --rm -it ghcr.io/sloretz/ros-peci1-gz-osrf:noetic-simulators-osrf gazebo --version
Gazebo multi-robot simulator, version 11.14.0
Copyright (C) 2012 Open Source Robotics Foundation.
Released under the Apache 2 License.
http://gazebosim.org

$ docker run --rm -it ghcr.io/sloretz/ros-peci1-gz-osrf:noetic-simulators gazebo --version
Gazebo multi-robot simulator, version 11.11.0
Copyright (C) 2012 Open Source Robotics Foundation.
Released under the Apache 2 License.
http://gazebosim.org
```

**Review by sloretz — APPROVED** [review]

**Comment by sloretz** [comment]
Thanks for the PR!

**Comment by peci1** [comment]
Now the question is whether you also want to provide the -osrf variants for ROS 2. However, I'm not using ROS 2 yet, so I'd rather not create the PR for that addition...

I also thought about announcing your repo on community.gazebosim.org . Or do you want to do the announcement yourself?

**Comment by peci1** [comment]
Done! https://community.gazebosim.org/t/up-to-date-docker-images-for-gazebo-ros-using-the-osrf-repo/2717 .

**Pull request merged**